### PR TITLE
Save to server [WIP]

### PR DESCRIPTION
**Pull request by ZNick1982**
enabled save to server options

### CODE REVIEW

**Comment by dmitry-mukhin** [comment]
So the issue was in jQuery version shipped in new WP?

**Comment by ZNick1982** [comment]
Inline insert issue were because UPLOADCARE_CDN_BASE was not defined.
The save to server option still have some issues, so PR will be updated.

File: changelog.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## Changelog
+**2.4.1** ([Download](http://downloads.wordpress.org/plugin/uploadcare.2.4.1.zip))
+* Minor bug fix for usage jQuery.
+* Test up to Wordpress 4.7.2
```
Comment (dmitry-mukhin) [inline changelog.md]: add a bit on copying to server

**Comment by ZNick1982** [comment]
Closing PR due.
The current feature will be added in new one.

**Pull request closed without merging**